custom_components/better_thermostat/climate.py

```diff
@@ -540,8 +550,8 @@ def precision(self):
 	@property
 	def target_temperature_step(self):
 		"""Return the supported step of target temperature."""
-		if self._target_temp_step is not None:
-			return self._target_temp_step
+		if self._TRV_target_temp_step is not None:
```

Comment:
Nope, this would return what the TRV supports, not what the user configured for Better Thermostat.